.github/workflows/build.yml

```diff
@@ -182,7 +182,7 @@ jobs:
           ${{ matrix.os }}-${{ matrix.compiler_cache }}
 
     - name: "Create build directory"
-      run: mkdir cmake_build
+      run: mkdir build
```

Comment:
Do all jobs have a new working directory?
I know an approach where you use subdirectories for different builds.
I this relevant here.
I use for instance a build/Debug folder and a build/RelWithDebugInfo folder.

Reply:
> Do all jobs have a new working directory?

On GitHub Actions, yes.

> I know an approach where you use subdirectories for different builds.
> I this relevant here.
> I use for instance a build/Debug folder and a build/RelWithDebugInfo folder

Every build starts in a clean container, so there is no reason to use different directories on GitHub Actions.

Reply:
Ok.

That's might be the issue for the packaging fail. Cpack alown fails.
You need to configure it with cmake first.

Reply:
The reason for the packaging fail was that the PPA step still used the `cmake_build` directory instead of `build`, but since that step was new it didn't caused a conflict. I amended the merge commit.